src/storage/file_io.h

```diff
@@ -335,7 +335,7 @@ struct fileio_backup_buffer
   int dtype;			/* Set to the type (dir, file, dev) */
   INT64 iosize;			/* Optimal I/O pagesize for backup device */
   INT64 count;			/* Number of current buffered bytes */
-  INT64 voltotalio;		/* Total number of bytes that have been either read or written (current volume) */
+  UINT64 voltotalio;		/* Total number of bytes that have been either read or written (current volume) */
   INT64 alltotalio;		/* total for all volumes */
```

Comment:
Changing only voltotalio type to UINT64, but leaving other similar fields are INT64 is hard to justify. Better to opt for one of next options:

  - change all four fields related to count/size as UINT64 and solve all signed/unsigned warnings that are caused by the change.
  - leave them as INT64 and fix signed/unsigned warnings related to voltotalio.